### DISCUSSION TITLE
docs/3.x/setup/configuration

### DISCUSSION THREAD

**Discussion by giscus** [saleor/saleor-docs#805] [General]
# docs/3.x/setup/configuration

Saleor follows the 12-factor approach, so you can configure Saleor using environment variables.

https://docs.saleor.io/docs/3.x/setup/configuration

**Comment by pitkes22** [comment]
One very important enviroment variable that is not mentioned on this page is **ENABLE_SSL**. This is essential when deploying to production behind a reverse proxy with HTTPS. It is used in multiple places when [constructing the absolute URL of the API](https://github.com/saleor/saleor/blob/1d21b8fd1a2cd196481216ab092fb858cafdf8da/saleor/core/utils/__init__.py#L31), especially for the `iss` parameter inside the JWT token. If `iss` is not set to the correct URL, it will cause authentication to not work with the new  [@saleor/auth-sdk](https://github.com/saleor/auth-sdk).
